PENDING.md

```diff
@@ -1,3 +1,10 @@
+## v0.24.0 PENDING 
+^--- PENDING wasn't purged on sdk v0.23.0 release.
+
+BREAKING CHANGES
+* Update to tendermint v0.23.0. This involves removing crypto.Pubkey, 
```

Comment:
Can we add the PR reference here (preferably with a link por favor)?

Reply:
But the SDK changed and that's important to note where/how it happened.

Reply:
I guess this is a downside of our pending system, the commit with all the changes can't be obtained from the file. I'll link the PR then.

Reply:
Yeah, I'd just would like to see a `CHANGELOG` and be able to go to the issue and/or PR that made the relevant changes.